### ISSUE TITLE
Can not use functions

### ISSUE THREAD

**Issue by KapitanPirula** [arcan1s/awesome-widgets#143]
Hi

I'm trying to use function inside the widget. If I try for example the one you use in https://arcanis.me/projects/awesome-widgets, I get this error:

``` javascript
${{
function colorCpu()
{
    if ($cpu > 90.0)
        return "<span style=\"color:#ff0000;\">$cpu</span>"
    else
        return "$cpu"
}
colorCpu()
}}
```

> [   511.729][WW][org.kde.plasma.awesomewidget][unknown] Uncaught exception at line 4 : "SyntaxError: Unexpected token `&'"

Even if I try this simple example, also get an error:
``` javascript
${{
$down - $up
}}
```

> [   796.231][WW][org.kde.plasma.awesomewidget][unknown] Uncaught exception at line 2 : "SyntaxError: Unexpected token `&'"

If I don't use variables, everything works:

```javascript
${{
function three()
{
    return 1+2;
}
three()
}}
```

**Comment by arcan1s** [comment]
:thinking:

**Comment by arcan1s** [comment]
i can confirm the bug, thank you, will check it

**Comment by arcan1s** [comment]
oh, i got it. Issue is that $cpu expands into something like `&nbsp;40` which is obviously not valid number. Not sure if it has been introduced by me or just qt has changed something (there is another issue which has been reported about nbsp in preview). Will fix it soon, thanks again

**Comment by KapitanPirula** [comment]
Thanks, I'll keep an eye out for updates

**Comment by arcan1s** [comment]
should be fixed with 3.4.2
